testsuite/pyproject.toml

```diff
@@ -0,0 +1,170 @@
+[build-system]
+requires = [
+  "setuptools >= 40.9.0",
```

Comment:
Is it correct that MDAnalysis itself (https://github.com/MDAnalysis/mdanalysis/pull/4424) doesn't require a minimum version?

Reply:
It probably does (because older versions don't support a lot of toml stuff), but we never set it. I might just set it to this value for package too - it's a good "you'll probably have trouble if you go below this" value.

Reply:
Updated in other PR - now we have a bare minimum pin of 40.9 which is the release that started supporting pyproject.toml.